### ISSUE TITLE
Incomplete install instructions

### ISSUE THREAD

**Issue by hmontenegro** [wegnerce/taxomias#3]
**TL;DR:**

Add instructions to create missing tables `tree` and `genomes` before running `setup_taxomias.py`.

**Long explanation:**

Following the README.md instructions to install taxomias I got an error at the `setup_taxomias.py` step:

```
Traceback (most recent call last):
  File "../bin/setup_taxomias.py", line 83, in <module>
    cursor.execute(command)    
sqlite3.OperationalError: no such table: tree
```

Probably due to the following commented out line:

`#cursor.execute("CREATE TABLE tree (taxid integer, name text, parent integer, rank text);")`

I uncommented the line and got:

```
Traceback (most recent call last):
  File "../bin/setup_taxomias.py", line 89, in <module>
    cursor.execute(command)
sqlite3.OperationalError: no such table: genomes
```

Now due to:

`#cursor.execute("CREATE TABLE genomes (taxid integer, path text);")`

Again, uncommented the line, and this time results in:

```
Traceback (most recent call last):
  File "../bin/setup_taxomias.py", line 79, in <module>
    cursor.execute("CREATE TABLE tree (taxid integer, name text, parent integer, rank text);")
sqlite3.OperationalError: table tree already exists
```

Final solution was to create both tables before running `setup_taxomias.py` and leave said ines commented out.

**Comment by wegnerce** [comment]
Thanks for your detailed comment. I will have a look into that, but you are right the problems are caused by the commented lines.

With both lines uncommented everything should run smoothly and it should not be necessary to create the tables manually beforehand.

Best

Carl-Eric
